### PR TITLE
Fp 936 lego skin global bootstrap variable changes

### PR DESCRIPTION
**Pull request by davidallardsb**
Jira: https://hxshortbreaks.atlassian.net/browse/FP-936

**What does this PR do? (please provide any background)**
This PR makes the Legoland version of the shortbreaks platform inline with the legoland brand.

**How can this be tested?**
Nothing to test here as the outcome of this PR will be testable in future tickets (availability page, upgrades page, etc).

**Any tech debt?**
Nope (although SBP will need CSS mappings to have correct styling applied. I will provide  information in relevant SBP pull requests).

#### What gif best describes how you feel about this work?

- [ ] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

---

**Reviewer**
- [ ] :+1:
- [ ] UXUI Team :+1:
- [ ] I don't think this PR needs any additional reviewers

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Ensured the release has the correct version bump.
- Checked all the work locally on pattern library.

### CODE REVIEW

File: brands/legoland/src/less/navbar.less [hunk 1]
```diff
@@ -31,6 +31,7 @@
 .navbar-inverse {
   background-image: url("@{brand-asset-path-images}nav-brick-pattern-light.png");
   background-size: auto 62px;
+  margin-bottom: @grid-gutter-width;
```
Comment (jodiedoubleday) [inline brands/legoland/src/less/navbar.less]: consider `@padding-base-vertical` here

File: brands/legoland/src/less/navs.less [hunk 2]
```diff
@@ -18,31 +18,23 @@
...
-  font-family: @headings-font-family;
-  > li {
-    margin-bottom: -2px;
-    > a {
-      background-color: @gray-lighter;
-      margin-right: -2px;
-      border: 2px solid @nav-tabs-border-color;
-      border-radius: @border-radius-small @border-radius-small 0 0;
+.nav.nav-tabs {
+  border-color: @white;
```
Comment (jodiedoubleday) [inline brands/legoland/src/less/navs.less]: you can change this is variables `@nav-tabs-border-color`

File: brands/legoland/src/less/navs.less [hunk 3]
```diff
@@ -18,31 +18,23 @@
...
-        border: 2px solid @nav-tabs-active-link-hover-border-color;
-        border-bottom-color: transparent;
-      }
+    &.active a {
+      border-radius: 0;
+      background: @white;
+      border-color: @white;
+    }
+    &:not(.active) a {
+      .btn;
```
Comment (jodiedoubleday) [inline brands/legoland/src/less/navs.less]: These may be variables, see - https://github.com/twbs/bootstrap/blob/v3.3.7/less/variables.less#L421

File: brands/legoland/src/less/panels.less [hunk 4]
```diff
@@ -1 +1,48 @@
...
+  background: @white;
+}
+
+.panel-info, .panel-success {
+  position: relative;
+  z-index: 1;
+  color: @white;
+  border: none;
+  overflow: hidden;
+  padding: 30px 15px 15px;
```
Comment (jodiedoubleday) [inline brands/legoland/src/less/panels.less]: please use padding variables - https://github.com/twbs/bootstrap/blob/v3.3.7/less/variables.less#L90

File: brands/legoland/src/less/panels.less [hunk 5]
```diff
@@ -1 +1,48 @@
...
+  // offsets background colour so it doesn't show behind lego brick pattern
+  &:before {
+    content: ' ';
```
Comment (jodiedoubleday) [inline brands/legoland/src/less/panels.less]: remove space between `''`

File: brands/legoland/src/less/panels.less [hunk 6]
```diff
@@ -1 +1,48 @@
...
+    position: absolute;
+    left: 0;
+    top: 600px;
```
Comment (jodiedoubleday) [inline brands/legoland/src/less/panels.less]: I'm not sure on this 600px value, could this cause us problems in the future?

File: brands/legoland/src/less/panels.less [hunk 7]
```diff
@@ -1 +1,48 @@
...
+    z-index: -1;
+    width: 100%;
+    height: 100%;
+    display: block;
+  }
+  border: none;
```
Comment (jodiedoubleday) [inline brands/legoland/src/less/panels.less]: move this to be with the rest of the parent styles

File: brands/legoland/src/less/panels.less [hunk 8]
```diff
@@ -1 +1,48 @@
...
+    background: rgba(255, 255, 255, 0.1);
+  }
+  label {
+    color: @white;
+  }
+}
+
+.panel-success {
+  background: url(@panel-success-bg-image) repeat-x;
+  &:BEFORE {
```
Comment (jodiedoubleday) [inline brands/legoland/src/less/panels.less]: Please use `before` instead of `BEFORE`

File: brands/legoland/src/less/panels.less [hunk 9]
```diff
@@ -1 +1,48 @@
...
+    background: @brand-success-brick-bg;
+  }
+}
+
+.panel-info {
+  background: url(@panel-info-bg-image) repeat-x;
+  &:BEFORE {
```
Comment (jodiedoubleday) [inline brands/legoland/src/less/panels.less]: as above

File: brands/legoland/src/less/progress-trackers.less [hunk 10]
```diff
@@ -5,6 +5,9 @@
 // --------------------------------------------------
 
 .progress-tracker {
+  background: @brand-blue-light;
+  border: 2px solid @brand-blue-light;
+  border-radius: 6px;
```
Comment (jodiedoubleday) [inline brands/legoland/src/less/progress-trackers.less]: use `@border-radius-large`

File: brands/legoland/src/less/variables.less [hunk 11]
```diff
@@ -47,29 +46,29 @@
 
 @font-family-sans-serif:  'Circular-Pro-Book', 'Noto Sans', Arial, 'Arial Unicode MS', sans-serif;
 
-@font-size-base:          16px;
-@font-size-large:         ceil((@font-size-base * 1.25)); // ~20px
-@font-size-small:         ceil((@font-size-base * 0.875)); // ~14px
+@font-size-base:          14px;
+@font-size-large:         16px; // ~20px
```
Comment (jodiedoubleday) [inline brands/legoland/src/less/variables.less]: comment doesn't match value

File: brands/legoland/src/less/variables.less [hunk 12]
```diff
@@ -47,29 +46,29 @@
...
+@font-size-small:         13px; // ~14px
```
Comment (jodiedoubleday) [inline brands/legoland/src/less/variables.less]: as above

File: brands/legoland/src/less/variables.less [hunk 13]
```diff
@@ -47,29 +46,29 @@
...
 
-@font-size-h1:            floor((@font-size-base * 2.25)); // ~36px
-@font-size-h2:            floor((@font-size-base * 1.5)); // ~24px
-@font-size-h3:            ceil((@font-size-base * 1.25)); // ~20px
-@font-size-h4:            ceil((@font-size-base * 1.125)); // ~18px
-@font-size-h5:            @font-size-base; // ~16px
-@font-size-h6:            ceil((@font-size-base * 0.875)); // ~14px
+@font-size-h1:            27px; // ~36px
```
Comment (jodiedoubleday) [inline brands/legoland/src/less/variables.less]: same with these...

File: brands/legoland/src/less/footer.less [hunk 14]
```diff
@@ -38,3 +38,15 @@
...
   }
 }
+
+.secure-footer {
+  padding: @padding-large-vertical*3;
+  background: @gray-lighter;
+  color: @gray-dark;
+  .tile-icons {
+    padding-top: @padding-large-vertical*2;
+    .tile-icons_image {
```
Comment (jodiedoubleday) [inline brands/legoland/src/less/footer.less]: this class is very SBPlatform specific and should not be put into brands.

File: brands/legoland/src/less/buttons.less [hunk 15]
```diff
@@ -15,3 +16,7 @@
     font-family: @font-family-sans-serif;
   }
 }
+
+.btn-gray {
```
Comment (davidallardsb) [inline brands/legoland/src/less/buttons.less]: I thought about this but a lot of the default buttons need to be red in SBP. Let's discuss over a call

File: brands/legoland/src/less/variables.less [hunk 16]
```diff
@@ -8,77 +8,88 @@
...
 
 @font-face {
  font-family: 'Circular-Pro-Bold';
- src: url("@{brand-asset-path-fonts}lineto-circular-pro-bold.woff");
+ src: url('@{brand-asset-path-fonts}lineto-circular-pro-bold.woff');
  font-weight: normal;
  font-style: normal;
 }
 
+@font-face {
```
Comment (davidallardsb) [inline brands/legoland/src/less/variables.less]: I'll show you over hangout

**Pull request merged**